addons/stock/report/report_stock.py

```diff
@@ -1,24 +1,23 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from openerp.osv import fields, osv
-from openerp.tools.sql import drop_view_if_exists
+from openerp import models, fields
+from openerp import tools
```

Comment:
No need to import separate for ``` tools ``` do like 
from openerp import fields, models, tools